src/openg2p_spar_mapper_api/services/mapper.py

```diff
@@ -194,9 +244,9 @@ async def update_mapping(session, single_update_request):
                 )
                 addl_info_keys = [info["name"] for info in addl_info_copy]
                 for info in single_update_request.additional_info:
-                    if info.name in addl_info_keys:
+                    if info["name"] in addl_info_keys:
```

Comment:
On line 242, add the type to `addl_info_copy` variable. I'm guessing it is `SingleUpdateRequest`. So that we can continue to use `info.name` instead of `info["name"]`